kafka-ui-react-app/src/components/Schemas/Details/LatestVersionItem.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { SchemaSubject } from 'generated-sources';
+import Editor from 'components/common/Editor/Editor';
+
+interface LatestVersionProps {
+  schema: SchemaSubject;
+}
+
+const LatestVersionItem: React.FC<LatestVersionProps> = ({
+  schema: { id, subject, schema, compatibilityLevel },
+}) => (
+  <div className="tile is-ancestor mt-1">
+    <div className="tile is-4 is-parent">
+      <div className="tile is-child">
+        <table className="table is-fullwidth">
```

Comment:
we already have version without bulma classes.

pls use styled components from this view https://github.com/provectus/kafka-ui/blob/master/kafka-ui-react-app/src/components/Schemas/Details/LatestVersion/LatestVersionItem.tsx